index.html

```diff
@@ -0,0 +1,80 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <link rel="reset" href="normalize.css">
+  <link rel="stylesheet" href="CSS/Portfolio_mystyles.css">
+  <meta name="viewport" content="width=device-width, initial-scale=1">
+  <link rel="preconnect" href="https://fonts.googleapis.com">
+  <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+  <link href="https://fonts.googleapis.com/css2?family=Lora:ital@0;1&family=Montserrat:wght@300&display=swap" rel="stylesheet">
+  <link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.8.1/css/all.css"   integrity="sha384-50oBUHEmvpQ+1lW4y57PTFmhCaXp0ML5d60M1M7uH2+nqUivzIebhndOJK28anvf" crossorigin="anonymous">
+  <title>Belinda Williams | Portfolio</title>
+</head>
+
+<body>
+  <nav class="dropdown">
+	<button class="dropbtn"><i class="fa fa-bars"></i></button>
+	  <div class="dropdown-content">
+		<a href="#home">HOME</a>
+	    <a href="#about">ABOUT</a>
+	    <a href="#work">WORK</a>
+	    <a href="#contact">CONTACT</a>
+	  </div>
+  </nav>
+  	
+  <header id="home" class="jumbotron">
+	<div class="image-wrapper"><img src="Images/Bath_Circus_1.jpg" alt="Circus, Bath"></div>
+	<section class="home-text">
+	  <h1>Portfolio</h1>
+	  <h2>BELINDA WILLIAMS</h2>
+    </section>
+  </header>
+
+<div class="arrow">
+    <a href="#about">
+        <i class="fas fa-chevron-down"></i>
+    </a>
+</div>
+
+<main>
+  <section id="about" class="about">
+    <img class="marker-img" src="Images/marker.png" alt="Location Marker">
+	<p class="location">Bath / London</p>
+	  <div class="personal">
+	    <p>Full Stack Software Developer and experienced Arts professional</p>
+	    <p>I turn <span>concepts</span> into <span>reality</span></p>
+      </div>
+  </section>
+	
+  <section id="work" class="work">
+    <h2>SELECTED WORK</h2>
+	<div class="project-container">
+	  <div class="project1">
```

Comment:
Each project has its own style (project1, 2, 3 in CSS). Do you intend to differentiate between how you style each project?